osbs/build/build_requestv2.py

```diff
@@ -444,6 +447,18 @@ def render_resource_limits(self):
             resources['limits'] = limits
             self.template['spec']['resources'] = resources
 
+    def set_deadline(self, build_type):
+        deadline_hours = 16
```

Comment:
is possible to make this configurable from a config instead of hardcoding it?

Reply:
Yes, done.